App.js

```diff
@@ -92,7 +90,7 @@ export default class App extends Component {
         true
       );
 
-      await RNSentiance.setValueForKey(userLinkFlag, "true");
+      await RNSentiance.enableNativeInitialization();
```

Comment:
It would be better to check if initialization was successful and only then enable native initialization.

Reply:
Hi @ralf601 At the time this line is being executed, the initialization is already completed successfully. If [the init here](https://github.com/sentiance/react-native-sentiance-example/blob/master/App.js#L88) fails, the function will return immediately and an error will be thrown and be caught [here](https://github.com/sentiance/react-native-sentiance-example/blob/master/App.js#L35). It is the way how the `await` keyword works in JS.

Reply:
Correct! we reject promise on init failure.